### ISSUE TITLE
singular they?

### ISSUE THREAD

**Issue by cxj** [emgrasmeder/about#3]
If I want to give you a pizza and I'm telling someone else about it, do I say "I gave _they_ a pizza?"  Or should I maybe use _them_?  Thanks.

**Comment by emgrasmeder** [comment]
Hi! Sorry, I missed this for so long. 

Yeah! Singular they is conjugated exactly like you'd expect. `I gave them a pizza`. 

I'll link to [pronoun.is/they](https://pronoun.is/they) from now on. Thanks for asking!

**Comment by cxj** [comment]
Thanks.  It's not always obvious when someone chooses personal pronouns if they intend to strictly follow conventional English grammar.

**Comment by emgrasmeder** [comment]
Indeed. And a lot of pronoun usage isn't "conventional" yet, since a lot of us are trail blazing and inventing new terms. 

If you didn't know, singular they has been used for hundreds of years (for example [Jane Austen made regular use of it](http://www.pemberley.com/janeinfo/austhlis.html#X2)).
